src/components/Tree/Node.tsx

```diff
@@ -53,6 +53,31 @@ type NodeProps = {
     onEditableSave?: (targetItemId: string, value: string) => void;
 };
 
+const setShowNodesInitial = (currentNode: DraggableItem<TreeNodeItem>, activeNodeIds?: NullableString[]): boolean => {
+    const hasActiveChildNodes = (node: DraggableItem<TreeNodeItem>, activeIds: NullableString[]): boolean => {
+        if (node.nodes) {
+            for (const childNode of node.nodes) {
+                if (activeIds.includes(childNode.id)) {
+                    return true;
+                }
+                if (hasActiveChildNodes(childNode, activeIds)) {
+                    return true;
+                }
+            }
+        }
```

Comment:
Also it would be great to simplify this function.

Reply:
Simplified it :-)